lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -52,17 +52,17 @@ public class IMRUCloseTaskTest : IMRUBrodcastReduceTestBase
         [Fact]
         public void TestTaskCloseOnLocalRuntime()
         {
-            int chunkSize = 2;
-            int dims = 50;
-            int iterations = 200;
-            int mapperMemory = 5120;
-            int updateTaskMemory = 5120;
-            int numTasks = 4;
-            string testFolder = DefaultRuntimeFolder + TestId;
+            const int chunkSize = 2;
```

Comment:
I was to trigger Jankins and made some changes on purpose. The change itself is valid anyway.